src/components/TokenSelect/index.tsx

```diff
@@ -77,14 +77,14 @@ export function TokenSelectDialog({ value, onSelect, onClose }: TokenSelectDialo
 
   if (!listHasTokens && isLoaded) {
     return (
-      <Dialog color="module" onClose={onClose}>
+      <Dialog color="module" onClose={onClose} data-testid="token-select-dialog">
```

Comment:
@zzmp I went with this pattern initially, since I'm more familiar with it, but the convention here is to query by user-accessible information first: https://testing-library.com/docs/queries/about#priority

Reply:
that makes sense, but it seems error-prone to query for the same string as the button text, and it makes it confusing to read. I'd prefer testid because of that, but could see the merits of either.